ee/desktop/notify/notify.go

```diff
@@ -4,23 +4,18 @@ import (
 	"github.com/go-kit/kit/log"
 )
 
-type DesktopNotifier struct {
-	logger       log.Logger
-	iconFilepath string
+type DesktopNotifier interface {
```

Comment:
Yeah, so that it can be injected into the desktop server -- https://github.com/kolide/launcher/pull/1017#discussion_r1115993124. I could definitely make a separate interface in that package, but I'm not sure I see the benefit?

Reply:
generally speaking, interfaces should be defined by the consumer and very slim.